CONTRIBUTING.md

```diff
@@ -187,25 +198,48 @@ Additional commits might be added to `master` branch between tagged-releases, so
 
 ### Setting-up Your Fork for Contributions
 
-If you intend to contribute contents to this project, you'll need to carry out some additional set-up steps in your local fork in order to be able to contribute to the _upstream_ repository:
+If you intend to contribute contents to this project, you'll need to first clone the repository on GitHub, create a local copy of your fork, and then carry out some additional set-up steps in your local fork in order to be able to contribute to the _upstream_ repository.
+You'll need to create a GitHub account, if you don't already have one.
+
+1. **FORK THE REPOSITORY** — Log-into GitHub and browse to the landing page of the official PureBasic repository:
+
+    - https://github.com/fantaisie-software/purebasic
+
+    then click on the <kbd>Fork</kbd> button that appears on the right side at beginning of the page (you need to be logged-in to see this option).
+    For more info on forking, see [GitHub Docs » Fork a repo][Fork a repo].
+
+2. **CLONE YOUR FORK LOCALLY** — You then need to _clone_ your fork locally, i.e. make a copy on your machine.
+Open up the Git Bash and type:
+
+    ```bash
+    $ git clone https://github.com/<your-GitHub-account>/purebasic.git
+    ```
+
+    replacing in the above command `<your-GitHub-account>` with whatever your GitHub account is (without brackets, exact letter casing).
+    For more info on how to clone a fork, see [GitHub Docs » Cloning a repository][Cloning a repository].
+
 
-1. **REMOTE SETTINGS** — You need to ensure that your fork is able to track the upstream repository by adding the `upstream` remote:
+3. **CONFIGURE THE UPSTREAM REMOTE** — You need to ensure that your fork is able to track the upstream repository by adding the `upstream` remote and _fetching_ it (i.e. updating its status information):
 
     ```bash
     $ git remote add upstream https://github.com/fantaisie-software/purebasic.git
+    $ git fetch upstream
     ```
 
-    From here-on your repository fork will recognize `upstream` as a reference to the main project (as opposed to the `origin` remote, which points to your fork on GitHub).
+    Henceforth, your repository fork will recognize `upstream` as a reference to the main project (as opposed to the `origin` remote, which points to your fork on GitHub).
+    For more info on how to add remotes, see [GitHub Docs » Adding a remote][Adding a remote].
 
-2. **TRACKING `devel`** — You then also need to ensure that your local fork of the repository contains a copy of the `devel` branch (which is not created automatically when you fork the repository):
+
+4. **TRACKING `master` BRANCH** — The default branch of the PureBasic repository is set to `devel`, not `master`, because it's main goal is development.
```

Comment:
This step seem to be optional. If that is the case you should make it explicit.

Reply:
It says "you _might_ also want to", which implies is not mandatory :

> So, you might also want to ensure that your local fork of the repository contains a copy of the `master `branch (which is not created automatically when you fork and clone) in order to gain access to the stable status of the project:

I think that the context is clear enough, especially if you take into account what was written before about the differences between `master `and `devel`.

I would you suggest to improve this text further?